src/default-test/java/nl/tudelft/jpacman/fuzzer/JPacmanFuzzer.java

```diff
@@ -0,0 +1,130 @@
+package nl.tudelft.jpacman.fuzzer;
+
+import nl.tudelft.jpacman.Launcher;
+import nl.tudelft.jpacman.board.Direction;
+import nl.tudelft.jpacman.game.Game;
+import nl.tudelft.jpacman.level.Player;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.RepetitionInfo;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * A basic fuzzer trying out random moves and writing logs in output files. Number of runs is a
+ * tunable parameter. Each run initiates the game and is played until PacMan dies.
+ *
+ * @author Azqa Nadeem
+ */
+@Disabled
+class JPacmanFuzzer {
+    private static final int RUNS = 5;
+    private static final String LOG_HEADER =
+        "isAlive, hasCollided, currentDirection, nextDirection, remainingPellets, score";
+
+    private Launcher launcher;
+    private BufferedWriter logWriter;
+
+    /**
+     * Sets and cleans up directory for storing logs.
+     *
+     * @throws IOException when file/folder can't be created.
+     */
+    @BeforeAll
+    static void setUpLogging() throws IOException {
+        File dir = new File("logs");
```

Comment:
Is there a (gradle) way to get this into the `out` folder?

Instead of removing log files, what about creating a time-stamped (System.currentTimeMillis) subfolder for each run?

Reply:
Ideally, this folder location algorithm needs to work locally in IntelliJ and Gradle, in our CI. However, given we are a bit out of time, I suggest we leave a variable here where students fill up a path to where they wanna store the logs.

Reply:
Let's do it 'time stamped' indeed and remove the file deletion.

`out/` is only used by IntelliJ I believe. Gradle uses `build/`. I don't think there's a nice way to put it in either one of these folders, without 'hardcoding it'. We can put them in the new behaviorial analysis directory @mauricioaniche created though.